classes/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, name: 'unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Since the `name` and `parent_permission` arguments uses colon notation, their assignments in the `super` method is somewhat different from what we are used to. Kindly implement the suggestion below

  ```
  super(age, name: name, parent_permission: parent_permission)
  ```